test/util/save_util_linux.cc

```diff
@@ -18,13 +18,23 @@
 
 #include "test/util/save_util.h"
 
+#if defined(__x86_64__) || defined(__i386__)
+#define SYS_TRIGGER_SAVE SYS_create_module
+#elif defined(__aarch64__)
+#define SYS_TRIGGER_SAVE SYS_finit_module
+#else
+#error "Unknown architecture"
+#endif
+
 namespace gvisor {
 namespace testing {
 
 void MaybeSave() {
   if (internal::ShouldSave()) {
     int orig_errno = errno;
-    syscall(SYS_create_module, nullptr, 0);
+    // We use it to trigger saving the sentry state when this syscall is called.
+    // Notice: This needs to be a valid syscall that is not used in any of the syscall tests.
```

Comment:
Please wrap C++ code and comments at 80 columns.

See the Google C++ style guide:
https://google.github.io/styleguide/cppguide.html

Reply:
Done.